Spanner/tests/System/SpannerPgTestCase.php

```diff
@@ -83,15 +82,24 @@ public static function setUpBeforeClass(): void
                     name varchar(1024) NOT NULL,
                     birthday date
                 )',
-                'CREATE ROLE ' . self::DATABASE_ROLE,
-                'CREATE ROLE ' . self::RESTRICTIVE_DATABASE_ROLE,
-                'GRANT SELECT ON TABLE ' . self::TEST_TABLE_NAME .
-                ' TO ' . self::DATABASE_ROLE,
-                'GRANT SELECT(id, name), INSERT(id, name), UPDATE(id, name) ON TABLE '
-                . self::TEST_TABLE_NAME . ' TO ' . self::RESTRICTIVE_DATABASE_ROLE,
             ]
         )->pollUntilComplete();
 
+        // Currently, the emulator doesn't support setting roles for the PG
+        // dialect.
+        if ((bool) getenv("SPANNER_EMULATOR_HOST") == false) {
```

Comment:
No need to cast to boolean or use `== false`, since these are all "truthy" comparisons anyway.
```suggestion
        if (!getenv("SPANNER_EMULATOR_HOST")) {
```